### ISSUE TITLE
GPU not detected

### ISSUE THREAD

**Issue by nedo99** [openvinotoolkit/openvino#16248]
##### System information (version)

- OpenVINO=> 2022.3
- Operating System / Platform =>Ubuntu 22.04
- Compiler => 11.3.0
- Problem classification => Device not detected

##### Detailed description
Hi,

I have this machine https://ark.intel.com/content/www/us/en/ark/products/196170/intel-nuc-12-enthusiast-kit-nuc12snki72.html with Ubuntu 22.04 (tested with Ubuntu 20.04 docker) and OpenVINO 2022.3. but I do not see the GPU in the devices list.

I am using oneAPI in parallel with tensorflow with intel extension and GPU is used properly.

##### Steps to reproduce

```
Core core;
    std::vector<std::string> availableDevices = core.GetAvailableDevices();
    slog::info << "Available devices: " << slog::endl;
    for (auto&& device : availableDevices)
        slog::info << device << slog::endl;
```
gives only CPU and GNA.
```
lspci | grep -i vga
00:02.0 VGA compatible controller: Intel Corporation Alder Lake-P Integrated Graphics Controller (rev 0c)
03:00.0 VGA compatible controller: Intel Corporation Device 5690 (rev 08)

```

##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [ ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by ilya-lavrenov** [comment]
@nedo99 

Please, provide output of `clinfo` command.

**Comment by nedo99** [comment]
After some debugging, it turned out that, either the driver was the issue, or my user was not part of the `render` group. `clinfo` was showing 0 platforms.

I did driver re-installation following this guide https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-arc.html#pre-installation-setting and it works now.

Thanks!
